spec/ewb/IEC61970/Base/Wires/StaticVarCompensator.yaml

```diff
@@ -0,0 +1,27 @@
+name: StaticVarCompensator
```

Comment:
todo: Again, not copied then modified from the base spec version. This is evident with the format of the description and the difference in units.

Reply:
Changed it up by Copying and modified.